charts/trento-server/charts/trento-web/Chart.yaml

```diff
@@ -15,4 +15,4 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.3.2
+version: 2.3.3-dev1
```

Comment:
This will change to `version: 2.3.3` when we have the next release ?

Reply:
Actually, since we are not changing the OBS label in this intermediate changes, let me try with `2.3.2+dev1`.
The reason of putting `-` was to enable a correct version number in OBS (`+` is not allowed), but in semver `-` means a smaller version.

As we don't know yet if we are going to relase `2.3.3` or `2.4.0` (or something else), using `+` is the way in semver

Reply:
The lint doesn't like `2.3.2+dev1`...
Going back to `2.3.3-dev1`